csm/core/services/users.py

```diff
@@ -213,6 +213,12 @@ async def get_user_list(self, limit, offset, sort_by, sort_dir):
 
         return [self._user_to_dict(x) for x in user_list]
 
+    async def get_user_count(self):
+        """
+        returns the count of existing users
+        """
+        return await self.user_mgr.count()
```

Comment:
Do you need to limit overall CSM users or only users with 'admin' role?

Reply:
@avrnv 

As discussed with @pawankumarsrivastava, requirement is to limit all the CSM users ( role[admin,manager,monitor]).